src/client/modules/my/summaryGraphs/summaryGraphs.ts

```diff
@@ -0,0 +1,59 @@
+import { LightningElement } from 'lwc';
+import { extractPlayerStats, getRawResults } from 'my/resultsFetcher';
+import type { GameResultsData, PlayerStatsAllGames } from 'my/resultsFetcher';
+
+// allows type-completion of the global-variable D3, which is assumed to already have been loaded (from script tag)
+declare global {
+    namespace d3 {}
+}
+
+const placeNumberToString = ['Zeroth', 'First', 'Second', 'Third', 'Fourth', 'Fifth', 'Sixth'];   // lazy way to convert; fine given we have a very small list
+
+type PlayersPerGame = Pick<GameResultsData, "game_label" | "player_num">;
```

Comment:
Pick: https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys
basically, an interface that's the same as GameResultsData, except only the `game_label` and `player_num` properties